extension.js

```diff
@@ -607,7 +601,7 @@ const ClipboardIndicator = Lang.Class({
         }
 
         notification.setTransient(true);
-        this._notifSource.notify(notification);
+        this._notifSource.showNotification(notification);
```

Comment:
This does not conflicts with another PR ?

Reply:
My bad, seems this has not been addressed yet. Another compatibility breaker :disappointed:

Reply:
The log said this API is deprecated and will be invalid soon.